mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -572,7 +574,8 @@ def request_wps_from_server(self, op_id):
                 "token": self.token,
                 "op_id": op_id
             }
-            r = requests.get(self.mscolab_server_url + '/get_operation_by_id', data=data, timeout=(2, 10))
+            url = urljoin(self.mscolab_server_url, "/get_operation_by_id")
```

Comment:
- leading slash

Reply:
I've removed the leading slash as suggested.